ExtLibs/UAVCAN/Extension.cs

```diff
@@ -39,19 +39,14 @@ public static object GetValue(this uavcan.uavcan_protocol_param_Value value)
             {
                 case uavcan.uavcan_protocol_param_Value_type_t.UAVCAN_PROTOCOL_PARAM_VALUE_TYPE_BOOLEAN_VALUE:
                     return value.union.boolean_value;
-                    break;
```

Comment:
the breaks should be there, as i dont want fallthough

Reply:
MP is a coding style that leaves the CASE-BREAK structure intact and describes the process.
I thought this was unnecessary, because I have never experienced that the RETURN process is not executed.